_posts/2020-07-06-concurrency-costs.md

```diff
@@ -451,7 +451,7 @@ class tls_counter {
     /* per-thread tls_counter object */
     static thread_local tls_counter tls;
 
-    /** add ourself to the counter list */
+    /** add ourselves to the counter list */
```

Comment:
Maybe this is archaic, but I think it is correct either way? In any case, I considered it when I wrote it... but yeah I'll change it.

Reply:
Maybe so? The singular/plural mixing of the word "ourself" makes it sound weird to me in general, but if I think about the royal "our" then it makes more sense.